Example/Tests/StylistTests.swift

```diff
@@ -51,6 +51,31 @@ final class StylistTests: XCTestCase {
             }
         }
     }
+
+    func testThemeing() {
+        let stylist = Stylist()
+
+        var didApplyGeneric = false
+        var didApplyThemed = false
+
+        stylist.addStyle(identifier: "element/atom") { view -> AnyView in
+            didApplyGeneric = true
+            return AnyView(view.foregroundColor(.red))
+        }
+
+        stylist.addStyle(identifier: "@dark/element/atom") { view -> AnyView in
+            didApplyThemed = true
+            return AnyView(view.foregroundColor(.blue))
+        }
+
+        stylist.currentTheme = Stylist.Theme(name: "@dark")
+
+        let stylable = Stylable(AnyView(Text("Test")), identifier: "element/atom")
+        _ = stylist.style(view: stylable, identifier: "element/atom")
+
+        XCTAssertFalse(didApplyGeneric)
+        XCTAssertTrue(didApplyThemed)
+    }
```

Comment:
Fancy also adding a test for @dark/atom overriding element/atom (or not, I don't know the expected precedence there!)

Reply:
yep, will work on that next. I suspect it should override in which case this PR won't work but I'll ping ya next week for a re-review